### ISSUE TITLE
lfqa_utils.py does not exist?

### ISSUE THREAD

**Issue by Weilin37** [facebookresearch/ELI5#24]
In the blogpost you mention that the lfqa_utils.py contain useful functions to run the model. I assume "qa_s2s_generate" is one of them. However, the link to the script leads to an empty page. Where can I find it? I'd like to try to run the pre-trained model.

**Comment by lokeshmeesala** [comment]
Hi Weilin37,
You can find this file in this location. https://github.com/huggingface/notebooks/blob/27f3013ea9d2534aca8fdfe8b7310ffc12458226/longform-qa/lfqa_utils.py#L375
